src/pages/config.tsx

```diff
@@ -76,13 +77,9 @@ const dnsJsonValidationFn = (value: string): Error | null => {
   }
 }
 
-const stringValidationFn = (value: string): Error | null => {
```

Comment:
validation function is optional, and we only use this for debug. removing and not passing validation function for debug input